### DISCUSSION TITLE
video thumbnails

### DISCUSSION THREAD

**Discussion by rejetto** [rejetto/hfs#499] [Announcements]
i just published version 3 of "thumbnails" plugin, which supports videos, limited to the formats supported by the browser itself.
must be enabled in plugin's options

<img width="328" alt="image" src="https://github.com/rejetto/hfs/assets/1367199/a9e392ff-d780-40de-b506-055e9b829d1c">

**Comment by THJ420** [comment]
where is the plugin?

**Comment by rejetto** [comment]
admin-panel > plugins > search online
it's called "thumbnails"
